WooCommerce/Classes/ViewRelated/Orders/Collect Payments/CollectOrderPaymentUseCase.swift

```diff
@@ -749,10 +751,10 @@ enum CollectOrderPaymentUseCaseError: LocalizedError {
             comment: "Error message when collecting an In-Person Payment and unable to update the order. %!$@ will " +
             "be replaced with further error details.")
 
-        static let unknownErrorWhileRefreshingOrderLocalizedDescription = NSLocalizedString(
-            "Unable to process payment. We could not fetch the latest order details. Please check your network " +
-            "connection and try again.",
-            comment: "Error message when collecting an In-Person Payment and unable to update the order.")
+        static let orderTotalChangedLocalizedDescription = NSLocalizedString(
+            "collectOrderPaymentUseCase.error.message.orderTotalChanged",
+            value: "Unable to process payment. Order total has changed since the beginning of payment.",
+            comment: "Error message when collecting an In-Person Payment and the order total has changed remotely.")
```

Comment:
```suggestion
        static let orderTotalChangedLocalizedDescription = NSLocalizedString(
            "collectOrderPaymentUseCase.error.message.orderTotalChanged",
            value: "Order total has changed since the beginning of payment. Please go back and check the order is " +
            "correct, then try the payment again.",
            comment: "Error message when collecting an In-Person Payment and the order total has changed remotely.")
```